gr-blocks/lib/add_const_ff_impl.cc

```diff
@@ -43,30 +43,26 @@ add_const_ff_impl::add_const_ff_impl(float k)
 {
 }
 
+void add_const_ff_impl::set_k(float k)
+{
+    d_k = k;
+    std::fill(d_k_copy.begin(), d_k_copy.end(), d_k);
+}
+
 int add_const_ff_impl::work(int noutput_items,
                             gr_vector_const_void_star& input_items,
                             gr_vector_void_star& output_items)
 {
     const float* iptr = (const float*)input_items[0];
     float* optr = (float*)output_items[0];
 
-    int size = noutput_items;
+    unsigned long input_size = (unsigned long)noutput_items;
```

Comment:
prefer the C++ `static_cast < unsigned long > (noutput_items);` over `(unsigned long)noutput_items;` ... they both do the same thing, of course , so this is style preference ...